src/types.ts

```diff
@@ -133,6 +144,7 @@ export interface AxisParentProps {
   paddingEnd?: number;
 
   flip?: boolean;
+  logScale?: boolean;
```

Comment:
To prepare for the third scale (time), we had in mind to use a `scale="log"` prop. See https://github.com/zakodium/react-plot/issues/78#issue-786806949